src/snowcli/cli/snowpark/compute_pool/commands.py

```diff
@@ -13,6 +15,22 @@
 )
 
 
+class InstanceFamily(str, Enum):
+    STANDARD_1 = "STANDARD_1"
+    STANDARD_2 = "STANDARD_2"
+    STANDARD_5 = "STANDARD_5"
+    HIGH_MEMORY_1 = "HIGH_MEMORY_1"
+    HIGH_MEMORY_2 = "HIGH_MEMORY_2"
+    HIGH_MEMORY_5 = "HIGH_MEMORY_5"
+    GPU_3 = "GPU_3"
+    GPU_5 = "GPU_5"
+    GPU_7 = "GPU_7"
+    GPU_10 = "GPU_10"
+
+
+INSTANCE_FAMILY_OPTIONS = [instance.upper() for instance in InstanceFamily]
```

Comment:
After changes, is this constant used anywhere?

Reply:
removed